db/migrate/20150903162623_assign_tenant.rb

```diff
@@ -38,9 +38,6 @@ class Vm < ActiveRecord::Base
   end
 
   def change
-    Tenant.connection.schema_cache.clear!
-    Tenant.reset_column_information
```

Comment:
Do you remember how to recreate the issue?  We should be able to write a test for it.